src/Controllers/ElementalAreaController.php

```diff
@@ -13,7 +13,7 @@
  */
 class ElementalAreaController extends LeftAndMain
 {
-    private static $url_segment = 'element-area';
+    private static $url_segment = 'elemental-area';
```

Comment:
If this isn't something we've changed recently, I wonder if it might be worth leaving it the way it was. I know we _can_ break stuff in this release, but if this is only for semantic reasons I'm not sure whether it's worth changing it in case people are relying on that URL being the way it was. Up to you though.

Reply:
This controller is new. `elemental` is more consistent with current naming conventions (of `ElementalArea`)

Reply:
It was a thing Guy added as the beginning of this work (the task this task depended on). I think it'll be safe to change, @ScopeyNZ himself said it should be changed :>